### PR TITLE
Newheaders

### PR DESCRIPTION
**Pull request by jdonszelmann**
New and improved headers, now with zero calories!

### CODE REVIEW

**Review by NULLx76 — DISMISSED** [review]
Reviewing my own code :thinking:

**Review by NULLx76 — DISMISSED** [review]

File: .editorconfig [hunk 1]
```diff
@@ -4,7 +4,7 @@ root = true
 end_of_line = lf
 insert_final_newline = true
 indent_style = space
-indent_size = 2
+indent_size = 4
```
Comment (dsluijk) [inline .editorconfig:7]: Why the change to indent size of 4? This was not discussed

File: Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "ipv8"
 version = "0.1.0"
-authors = ["Dany Sluijk <dany@atlasdev.nl>"]
+authors = ["jonay2000 <jonabent@gmail.com>", "NULLx76 <victor@xirion.net>"]
```
Comment (dsluijk) [inline Cargo.toml]: Please at least keep me in the list...

File: hooks/pre-commit [hunk 3]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```
Comment (dsluijk) [inline hooks/pre-commit]: This file has been added in a different PR

**Review by dsluijk — CHANGES_REQUESTED** [review]

File: .editorconfig (hunk 1 above)
Comment (NULLx76) [reply .editorconfig:7]: It was in the #37 pull request, I just forgot to edit this file there.

File: hooks/pre-commit (hunk 3 above)
Comment (NULLx76) [reply hooks/pre-commit]: yes, and we've just merged that branch to here. So #40 is blocking for this pull.

**Review by dsluijk — APPROVED** [review]

**Review by NULLx76 — APPROVED** [review]
